rules/warn-then-call/rule.md

```diff
@@ -0,0 +1,81 @@
+---
+type: rule
+title: Calling - Do you check your contact is free before calling them?
+uri: warn-then-try-calling-them
```

Comment:
```suggestion
uri: warn-then-call
```

Reply:
I thought I'd already changed this. Hmm. Thanks for the pickup :)